README.md

```diff
@@ -1,11 +1,11 @@
-# Slash Mercury Parser
+# Parser
 
 This is a [serverless](https://serverless.com/) Slack Bot that uses Postlight's [mercury parser](https://github.com/postlight/mercury-parser) to display the parsed content directly in your slack channel! So instead of sending a link to a certain article that will open in a new browser window, you will be sending the article as a readable post directly.
```

Comment:
should we update this now, or wait until github rename?

Reply:
good question! i can't think of any reason to wait but i'd be happy to revert this change for the time being, if necessary

Reply:
let's update all the slack stuff in [here](https://github.com/postlight/slash-mercury-parser/issues/6)